lib/src/widgets/adw/switch.dart

```diff
@@ -0,0 +1,675 @@
+// Copyright 2014 The Flutter Authors => BSD-3 License
+// Copyright 2022 Gtk-Flutter Authors => LGPL-3 License
+
+import 'dart:ui' show lerpDouble;
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter/services.dart';
+import 'package:libadwaita/libadwaita.dart';
+
+// Examples can assume:
+// bool _lights = false;
+// void setState(VoidCallback fn) { }
+
+/// An gtk-style switch.
```

Comment:
Very small grammar error: `A gtk-style switch.`

Also present elsewhere in the file.